verifier/verifier.go

```diff
@@ -478,8 +500,10 @@ func executePlugin(ctx context.Context, installedPlugin plugin.Plugin, trustPoli
 		TrustPolicy:  policy,
 		PluginConfig: pluginConfig,
 	}
-
-	return installedPlugin.VerifySignature(ctx, req)
+	logger.Debugf("plugin VerifySignatureRequest: %+v", req)
+	resp, err := installedPlugin.VerifySignature(ctx, req)
+	logger.Debugf("plugin VerifySignatureResponse: %+v", resp)
```

Comment:
`resp` might not be reliable if `err` is not `nil`.

Reply:
moved to the function caller.